src/Core/Enums/DatePickerRenderingMode.cs

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------------------------------------
+// This file is licensed to you under the MIT License.
+// ------------------------------------------------------------------------
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary />
+public enum DatePickerRenderingMode
+{
+    /// <summary />
+    FluentUI,
+
+    /// <summary />
+    Browser,
```

Comment:
I think we should use `Native` instead of `Browser` (better suited when, for example, using the components in Blazor Hybrid.

Also, a bit in doubt about `...RenderingMode` Could be confused with InteractiveServer/Wasm/Auto

Reply:
I rename the parameter to `RenderStyle` and the enum value to `Native`